util/yaml/yaml.go

```diff
@@ -20,8 +20,10 @@ package yaml
 import (
 	"bufio"
 	"bytes"
+	"github.com/MakeNowJust/heredoc"
```

Comment:
not sure where the current CNCF guidelines are for the use of third party projects in the codebase but we should make sure this one is okay from a licensing perspective, etc.

Reply:
The license seems MIT

Reply:
👍

Reply:
this import is already used in clusterctl...